.github/workflows/blackduck.yml

```diff
@@ -3,10 +3,12 @@ name: Blackduck Scan
 on:
   push:
     branches:
-      - integration/main
+      - "integration/main"
```

Comment:
Be careful here. 

1. For every new version, you will have to manually make a new clone of the existing version to that new version, or the pipeline will update to Blackduck with none of the existing review data, resulting in every import 150+ having to be reviewed.

Reply:
yes understood, we will be making release from integration/main only, the scans here are to detect for every PR made, rather than having it done at the end all together. In this way, we have to clone only for one branch which is integration/main in blackduck